app/assets/stylesheets/application.scss

```diff
@@ -55,6 +55,11 @@ body {
 
 }
 
+.timeline_time_frame{
```

Comment:
can we move this to the `app/assets/stylesheets/dm/pages/_practice.scss` file since it is specifically styles for the timeline?